docs/tutorial/part-three/index.md

```diff
@@ -82,20 +82,20 @@ export const { scale, rhythm, options } = typography
 export default typography
 ```
 
-4. Start the development server.
+4. डेवलपमेंट सर्वर शुरू करें।
 
 ```shell
 gatsby develop
 ```
 
-Once you load the site, if you inspect the generated HTML using the Chrome developer tools, you’ll see that the typography plugin added a `<style>` element to the `<head>` element with its generated CSS:
+एक बार साइट लोड करने के बाद, यदि आप क्रोम डेवलपर टूल का उपयोग करके जेनरेट किए गए HTML का निरीक्षण करते हैं, तो आप देखेंगे कि टाइपोग्राफी प्लगइन ने `<style>` तत्व को `<head>` तत्व के साथ अपने उत्पन्न सीएसएस में जोड़ा:
```

Comment:
`सीएसएस` => `CSS`

I think we should not translate CSS, what do you think? 
We have translated this in few other pages, we will finalize and revert those

Reply:
Yup. I agree